report/src/main/kotlin/com/hartwig/actin/report/interpretation/MolecularTestInterpretation.kt

```diff
@@ -0,0 +1,31 @@
+package com.hartwig.actin.report.interpretation
+
+import java.time.LocalDate
+
+data class MolecularTestInterpretation(
```

Comment:
These classes and the following all seem only applicable to IHC tests but names indicate they are for all molecular tests.